src/components/CropSelector/CropTable.js

```diff
@@ -184,7 +185,7 @@ const CropTableComponent = (props) => {
             </Button>
           </div>
         </TableCell>
-      </Fragment>
+      </>
```

Comment:
we should keep the Fragment tag here.  Empty tags are ok when we are only trying to appease the JSX requirement for all JSX to be under a single tag.  but we want to be intentional as often as possible.  removing a fragment tag for an empty tag would be considered regression.

Reply:
Okay, but doesn't an empty tag imply a fragment? then why are we using the keyword 'Fragment' explicitly?